generator/_config.yml

```diff
@@ -0,0 +1,50 @@
+#auto: true
```

Comment:
why did you revert _config.yml deletion in https://github.com/cfengine/documentation/pull/3448/commits/6b1e0df504e23e72ce7970026affc736997c0ac9 ?

I imagine you might squash some of these commits so probably the commit message doesn't matter.